messaging.rst

```diff
@@ -349,6 +349,7 @@ For the simplest Alice - Bob example:
 - Alice creates a new transfer with:
     * transferred_amount = ``current_value``
     * lock = ``Lock(n, hash(secret), expiration)``
+    * locked_amount = ``updated value containing the lock amount``
     * locksroot = ``updated value containing  the lock``
```

Comment:
Not related to your PR, but there's a space to much between *containing* and *the*.